README.md

```diff
@@ -208,6 +208,12 @@ Allows custom monitoring events to be added into the Node Application Metrics ag
 * `type` (String) the name you wish to use for the data. A subsequent event of that type will be raised, allowing callbacks to be registered for it.
 * `data` (Object) the data to be made available with the event. The object must not contain circular references, and by convention should contain a `time` value representing the milliseconds when the event occurred.
 
+### appmetrics.writeSnapshot([filename],[callback])
```

Comment:
I would call this "writeHeapSnapshot", as-is, its not possible to guess the purpose of the API from the name - it could be a snapshot of many things.